### DISCUSSION TITLE
Remote debugging using VS Code requires exporting DISPLAY in launch.json

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#16146] [General]
Making some test in Linux I have noticed that to be able to remote debug the application it is needed to export DISPLAY in the environment section of launch.json.

I found something similar for Visual Studio but not for V.S. Code here:

https://platform.uno/docs/articles/get-started-with-linux.html?tabs=ubuntu1804#setting-for-windows-and-wsl

So this should be added to the documentation.

**Comment by jeromelaban** [comment]
Hello @mcNets, I was modifying the docs for Linux and I added it, it will be in the next release of the docs. Thanks for the update!

**Reply by mcNets** [reply]
Ok @jeromelaban, there is an open pending PR.

https://github.com/unoplatform/uno/pull/16147

**Reply by jeromelaban** [reply]
Will take a look, thanks!
